.github/workflows/buildAndTest.yml

```diff
@@ -0,0 +1,71 @@
+# Copyright 2022 The StableHLO Authors.
+
+name: Build and Test
+
+on:
+  pull_request:
+    branches: [ main ]
+  push:
+    branches: [ main ]
+  schedule:
+    # Always regenerate once every day
```

Comment:
Wdym by "regenerate"?

Reply:
Modified comment. "Run this workflow once a day." 

Regenerate doesn't make sense unless we're publishing releases or something.